src/InputGroupButton.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import Button from './Button';
+
+const propTypes = {
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  children: PropTypes.node,
+  groupClassName: PropTypes.any,
```

Comment:
Yeah, definitely. I like the `props` since `className` is just a prop, one could do `props={{group: {className: 'yo'}}}` to get that there without a special prop just for it.
I can change this in a future PR to match that pattern if you want to go that way.